records/config.go

```diff
@@ -202,6 +208,10 @@ func (c *Config) initSOA() {
 
 func (c Config) log() {
 	// print configuration file
+	zoneResolversJSON, err := json.Marshal(c.ZoneResolvers)
+	if err != nil {
+		zoneResolversJSON = []byte("error")
```

Comment:
nit: it might be helpful to actually see the error message here. probably not likely that we'd every see one